src/main/java/com/google/api/codegen/nodejs/NodeJSGapicContext.java

```diff
@@ -353,7 +359,8 @@ private String returnTypeComment(Method method, MethodConfig config) {
           + " for write() method.";
     }
     return callbackMessage
-        + "\n@returns {Promise} - The promise which resolves to the response object.\n"
+        + "\n@returns {Promise} - The promise which resolves to an array\n"
+        + "  of the same parameters to the callback except for the Error.\n"
```

Comment:
nit: This line seems a bit hard to understand. I think you should explicitly state what the parameters of the callback are.